lib/unimatrix/authorization/filters/requires_policies.rb

```diff
@@ -70,29 +69,30 @@ def self.included( controller )
       controller.extend( ClassMethods )
     end
 
-    def policies=(attributes)
+    def policies=( attributes )
       @policies = attributes
     end
 
     def policies
-      @policies ||= begin
```

Comment:
What I'm saying is that not all use the requires_policies filter. Some use the has_permission? method to check. 

For example PropertiesController#create, does not use the before filter because its a PUT to /properties with no property_id in the params. I'll admit this is kind of weird, it should be a POST to /properties/:property_id, but this will all change anyway with your regent work.

Also the ArtifactsController uses it to check if the user has create and publishing permissions - though in the artifacts controller policies would have already been set, but you may just want to check that this gem update does not break that check.

"Both controller_name and realm_uuid are undefined here so this couldn't have worked." I know this works. In archivist realm_uuid would be defined at this point and controller_name is a rails helper method. 

Perhaps this can be refactored better but without this, updating properties in portal will definitely break.